index.html

```diff
@@ -63,6 +60,171 @@ <h2>I'am a Software Developer</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="projectlinks">
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+    
+            <button class="btn">See Project</button>
+
+        </div>
+            </div>
+        </div>
+
+        <div class="project project6">
+            <div class="content">
+                <h2 class="contenthead">Professional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="projectlinks">
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+            <button class="btn">See Project</button>
+
+        </div>
+            </div>
+        </div>
+        <div class="project project7">
+            <div class="content">
+                <h2 class="contenthead">Professional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul class="projectlinks">
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+            <button class="btn">See Project</button>
+
+        </div>
+            </div>
+        </div>
+               <!--ABOUT SECTION-->
+ <div class="aboutsection">        
+<div class="about-part">
+<h2 class="aboutmehead">About me</h2>
+<p class="aboutpara">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+<button type="button" class="buttn">Get My Resume</button>
+<article class="container2">
+    <div class="aboutinfo">
+</div>
+</div>
+<hr class="line">
  </div>
+<div class="tools">
+    <div class=" tool-list">
+        <h3 class="toolhead">Languages<img src="Images/Rectangle 55.png" </h3>
+        <ul class="lists">
+            <li>Javascript</li>
+            <li>css</li>
+            <li>html</li>
+            <li>Ruby</li>
+        </ul>
+    </div>
+    <div class=" tool-list">
+        <h3 class="toolhead">Frameworks<img src="Images/Rectangle 56.png" </h3>
+        <ul class="lists">
+            <li>Ruby on rails</li>
+            <li>Bootstrap</li>
+            <li>RSpec</li>
+            <li>Capybara</li>
+            <li>Selenium</li>
+        </ul>
+    </div>
+
+    <div class=" tool-list">
+        <h3 class="toolhead">Skills<img src="Images/Rectangle 57.png" </h3>
+        <ul class="lists">
+            <li>Codekit</li>
+            <li>GitHub</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+        </ul>
+    </div>
+</div>
```

Comment:
- Kindly wrap this block of code in a `section` tag. It also represents a generic standalone section of the document. :+1:  :smiley:  :smiley: 
- Kindly wrap the technology categories ("Languages", "Frameworks" and "Skills") inside a `ul`, each of them in a `li`. This is part of the requirements of the project. :scroll: 
![image](https://user-images.githubusercontent.com/98363075/192100197-bbd9abd4-3bdb-4331-8bf3-b66fb99edafe.png)